MdePkg/Include/IndustryStandard/IpmiSerial.h

```diff
@@ -0,0 +1,60 @@
+/** @file
+  IPMI Serial Definitions
+
+  Copyright (c) 2024, ARM Limited. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+  @par Revision Reference:
+    - IPMI Specification
+      Version 2.0, Rev. 1.1
+
+  https://www.intel.com/content/www/us/en/products/docs/servers/ipmi/ipmi-second-gen-interface-spec-v2-rev1-1.html
+**/
+
+#ifndef IPMI_SERIAL_H_
+#define IPMI_SERIAL_H_
+
+///
+/// IPMI Serial Escaped Character Definition
+/// Section 14.4.1 & 14.4.2
+///
+#define BM_START      0xA0
+#define BM_STOP       0xA5
+#define BM_HANDSHAKE  0xA6
+#define BM_ESCAPE     0xAA

```

Comment:
Could you please replace BM_ with BASIC_MODE_

Reply:
Acked.